src/Components/Web/src/Forms/InputFile/BrowserFileExtensions.cs

```diff
@@ -26,12 +26,13 @@ public static class BrowserFileExtensions
         /// <param name="format">The new image format.</param>
         /// <param name="maxWidth">The maximum image width.</param>
         /// <param name="maxHeight">The maximum image height</param>
+        /// <param name="quality">The quality of the image</param>
         /// <returns>A <see cref="ValueTask"/> representing the completion of the operation.</returns>
-        public static ValueTask<IBrowserFile> RequestImageFileAsync(this IBrowserFile browserFile, string format, int maxWidth, int maxHeight)
+        public static ValueTask<IBrowserFile> RequestImageFileAsync(this IBrowserFile browserFile, string format, int maxWidth, int maxHeight, double? quality = null)
```

Comment:
You'll need to add a new overload that accepts the quality parameter. Adding an additional parameter to an existing method is a binary breaking change.